From b6f16f656bb8683f70ce8ce19791d594fce4163b Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 15 Jan 2021 19:54:08 +0000
Subject: [PATCH] add dependency on opus in webcodecs

webcodecs uses opus, but dependency is missing. With unbundled
opus library build fails, because include path is incomplete.
---
 third_party/blink/renderer/modules/webcodecs/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/renderer/modules/webcodecs/BUILD.gn b/third_party/blink/renderer/modules/webcodecs/BUILD.gn
index 2c407ec..0474bfe 100644
--- a/third_party/blink/renderer/modules/webcodecs/BUILD.gn
+++ b/third_party/blink/renderer/modules/webcodecs/BUILD.gn
@@ -63,6 +63,7 @@ blink_modules_sources("webcodecs") {
     "//media/mojo/clients",
     "//media/mojo/mojom",
     "//third_party/libyuv:libyuv",
+    "//third_party/opus",
   ]
   if (media_use_openh264) {
     deps += [ "//third_party/openh264:encoder" ]
-- 
2.26.2

